_data/other.yml

```diff
@@ -220,6 +220,13 @@ websites:
       sms: Yes
       software: Yes
 
+    - name: Monster
+      url: http://www.monster.com
```

Comment:
@stephengroat, It looks like Monster supports https. Would it be possible to update the URL to reflect that?